### PR TITLE
Tu brand

### PR DESCRIPTION
**Pull request by yceballost**
There are some inconsistencies with current web skin in loading bars, but dark mode is not important now and is expected to fix in web in the next token update

### CODE REVIEW

File: tokens/tu.json [hunk 1]
```diff
@@ -0,0 +1,1414 @@
...
+        "type": "color"
+      },
+      "red70": {
+        "value": "#8E3629",
+        "type": "color"
+      },
+      "red80": {
+        "value": "#5B231A",
+        "type": "color"
+      },
```
Comment (aweell) [inline tokens/tu.json:1343]: Maybe purples are missing here?

purple: '#7274E7',
    purple10: '#F1F1FD',
    purple40: '#9C9EEE',
    purple70: '#5051A2',

File: tokens/tu.json [hunk 2]
```diff
@@ -0,0 +1,1414 @@
...
+      "type": "color",
+      "description": "grey3"
+    },
+    "buttonPrimaryBackgroundHover": {
+      "value": "{palette.grey3}",
+      "type": "color",
+      "description": "grey3"
+    },
+    "buttonPrimaryBackgroundInverseSelected": {
+      "value": "{palette.primary80}",
```
Comment (aweell) [inline tokens/tu.json]: In web this is:

`buttonPrimaryBackgroundInverseSelected: palette.blue70,`

**Review by aweell — CHANGES_REQUESTED** [review]

**Comment by aweell** [comment]
https://mistica-design.vercel.app/tokens-map?branch=tu-brand&skin=tu&tokenType=color&activeColor=undefined

File: tokens/tu.json (hunk 2 above)
Comment (yceballost) [reply tokens/tu.json]: yes, this is a mistake in web. Blue70 in dark mode is wrong
image

But primary80 could be wrong too, the correct is `grey3`

changed!

File: tokens/tu.json (hunk 1 above)
Comment (yceballost) [reply tokens/tu.json:1343]: purple doesn't exist in Tu brand palette 🤔

Comment (aweell) [reply tokens/tu.json:1343]: They will be erased when the action is triggered then, guess no prob here

**Review by aweell — APPROVED** [review]
LGTM

**Pull request merged**